From 553d4d70d128e78fd12031f5109d0164088c784f Mon Sep 17 00:00:00 2001
From: Elijah Newren <newren@gmail.com>
Date: Tue, 21 Mar 2023 06:25:55 +0000
Subject: [PATCH 1077/2871] treewide: remove unnecessary inclusion of gettext.h

Looking at things from the opposite angle of the last patch, we had a
few files that were including gettext.h and perhaps needed it at some
point in history, but no longer require it.  Remove the include.

Signed-off-by: Elijah Newren <newren@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 progress.c                          | 1 -
 t/helper/test-parse-pathspec-file.c | 1 -
 t/helper/test-progress.c            | 1 -
 t/helper/test-run-command.c         | 1 -
 4 files changed, 4 deletions(-)

diff --git a/progress.c b/progress.c
index 9b33a2df32..44c784d75f 100644
--- a/progress.c
+++ b/progress.c
@@ -10,7 +10,6 @@
 
 #define GIT_TEST_PROGRESS_ONLY
 #include "cache.h"
-#include "gettext.h"
 #include "progress.h"
 #include "strbuf.h"
 #include "trace.h"
diff --git a/t/helper/test-parse-pathspec-file.c b/t/helper/test-parse-pathspec-file.c
index 71d2131fba..89ecefd1cd 100644
--- a/t/helper/test-parse-pathspec-file.c
+++ b/t/helper/test-parse-pathspec-file.c
@@ -1,7 +1,6 @@
 #include "test-tool.h"
 #include "parse-options.h"
 #include "pathspec.h"
-#include "gettext.h"
 
 int cmd__parse_pathspec_file(int argc, const char **argv)
 {
diff --git a/t/helper/test-progress.c b/t/helper/test-progress.c
index 6cc9735b60..66acb6a06c 100644
--- a/t/helper/test-progress.c
+++ b/t/helper/test-progress.c
@@ -19,7 +19,6 @@
  */
 #define GIT_TEST_PROGRESS_ONLY
 #include "test-tool.h"
-#include "gettext.h"
 #include "parse-options.h"
 #include "progress.h"
 #include "strbuf.h"
diff --git a/t/helper/test-run-command.c b/t/helper/test-run-command.c
index b0d041ec5f..c0ed8722c8 100644
--- a/t/helper/test-run-command.c
+++ b/t/helper/test-run-command.c
@@ -16,7 +16,6 @@
 #include "string-list.h"
 #include "thread-utils.h"
 #include "wildmatch.h"
-#include "gettext.h"
 
 static int number_callbacks;
 static int parallel_next(struct child_process *cp,
-- 
2.39.3 (Apple Git-145)

